widget_driver/pubspec.lock

```diff
@@ -5,42 +5,42 @@ packages:
     dependency: transitive
     description:
       name: async
-      url: "https://pub.dartlang.org"
+      url: "https://mobile20unpubfrontdoor.azurefd.net"
```

Comment:
I am not sure if we want to commit these urls :) 
It is probably better to keep these as the public url to pub dev.. and not the bmw internal proxy ones. :)